### ISSUE TITLE
Use proper path when extending nested templates

### ISSUE THREAD

**Issue by marcinkrzeminski** [xfiveco/generator-chisel#291]
There seems to be a bug in the components/tease-post.twig. It extends tease.twig like this 

```
{% extends "tease.twig" %}
```

This seems not to work which leads to two problems:

1. This file won't be used while using loop like this:
```
{% for post in posts %}
    {% include ['components/tease-'~post.post_type~'.twig', 'components/tease.twig'] %}
{% endfor %}
```

Since the file is not there the fallback one will be used.

2. If no fallback templated passed and used the explicit one: ``{% include 'components/tease-'~post.post_type~'.twig' %}`` it throws an error.

The fix is pretty simple just make sure to use relative path to the theme root even if the extending files are inside the same child directory: ``{% extends "components/tease.twig" %}``

Could you check if the error occurs on your side as well? If confirmed I can work on this fix.

P.S. I'm using latest Timber version.

**Comment by jakub300** [comment]
Fixed in 837d8fd09383b3e0b865d4c620912f76c56ecce3?

**Comment by marcinkrzeminski** [comment]
Yes :(. Didn't check that before posting an issue. Closing.
